content/pages/sponsors.md

```diff
@@ -0,0 +1,17 @@
+Title: Sponsorship
+
+Occasionally companies will help us run events by providing us with funding, currently we are working
+on the following projects:
+
+### Technical Workshop Series with BlackBerry
+
+Currently we are running a Technical Workshop Series, which will consist of three
+technical skills workshops over the next three terms.
+
+These events are generously sponsored by [BlackBerry](http://ca.blackberry.com)!
+BlackBerry has agreed to fund our three events and we are very excited to 
+work with them!
+
+To view all the event posts for our technical workshops, click 
+[here](http://wics.uwaterloo.ca/tags/blackberry). In Fall 2015 we ran our 
+Arduino Workshop, and we plan on running one more this term.
```

Comment:
I've already done a lot of --amend D: ugly git history is ugly. We can make this page better when we have more time - anna also wanted to add a blurb about what kind of sponsorship we're looking for and stuff.